<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GNG3DIWindow</class>
 <widget class="QMainWindow" name="GNG3DIWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: #ffffff;
background-color: #333333;
padding: 0;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #666666;
border-bottom: 1px solid #262626;</string>
   </property>
   <widget class="QMenu" name="menuGNG3DI">
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #262626;

selection-background-color: #333333;
selection-color: #ffffff;</string>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;GNG3DI</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actionLoad_mesh"/>
    <addaction name="saveMesh"/>
    <addaction name="actionSwap_mesh"/>
    <addaction name="resetMesh"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuParameters">
    <property name="title">
     <string>Para&amp;meters</string>
    </property>
   </widget>
   <addaction name="menuGNG3DI"/>
   <addaction name="menuParameters"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: #666666;
border-top: 1px solid #262626;</string>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings ...</string>
   </property>
  </action>
  <action name="actionOpen_Folder">
   <property name="text">
    <string>&amp;Open Folder ...</string>
   </property>
  </action>
  <action name="actionLoad_mesh">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Load mesh ...</string>
   </property>
   <property name="iconText">
    <string>Load mesh ...</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="action">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save &amp;mesh ...</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="saveMesh">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sa&amp;ve mesh ...</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="resetMesh">
   <property name="text">
    <string>&amp;Reset</string>
   </property>
   <property name="iconText">
    <string>Reset</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSwap_mesh">
   <property name="text">
    <string>Swap mesh ...</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
